src/shared/styles/components/top-bar.css

```
@@ -238,9 +249,11 @@
   align-items: center;
   justify-content: center;
 
-  padding: 0.5rem;
-  min-width: 40px;
-  min-height: 40px;
+  padding: 0;
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)